client/src/constants/routes.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-useless-escape */
```

Comment:
I think we should keep this rule. Can be misleading otherwise.

Reply:
It triggers on escaping the `.` in our RegExp, I don't see any better way around it.